docs/hoon/library/1c.md

```diff
@@ -9,243 +9,295 @@ navhome: /docs/
 
 Blocksize
 
-Atom representing a blocksize, by convention expressed as a power of 2.
+Atom representing block size. A block of size `a` has a bitwidth of `2^a`.
 
-Source
-------
+#### Source
 
-    ++  bloq  @                                            ::  blockclass
+    ++  bloq  @                                         
 
-Examples
---------
+#### Examples
 
-    > :: ++met measures how many bloqs long an atom is
-    > (met 3 256)
+    ::++met measures how many a-bloqs long an atom is. In other words, how
+    many bloqs of size a would be needed to represent b.
+
```

Comment:
Neatness issue---before we decided to add 'discussion' sections, this sort of comment got thrown in to the examples.  I think it would be better to remove lengthy comments from examples and add that commentary to 'discussion'.  Very brief comments seem fine in the examples, but long ones look sloppy.  The longest ones require scrolling right to read the whole thing, which is bad UX as well.